docs/_resources/examples/inspiration/dell/_brand.yml

```diff
@@ -0,0 +1,174 @@
+## This is copy of /docs/inspiration/brand-guidelines/dell/_brand-dell.yml
+## Eventually we should ensure there's a single source of truth for this data
```

Comment:
Could `docs/_resources/examples/inspiration/dell/index.qmd` have a `brand` key that points to the brand.yml file in a different directory?

Reply:
(I agree, we should try not to need copies of these files)

Reply:
> Could docs/_resources/examples/inspiration/dell/index.qmd have a brand key that points to the brand.yml file in a different directory?

Unfortunately not; Quarto doesn't like you pulling files out of the _quarto.yml root. There might be other ways to go about it, but not this directly.

Reply:
What if we keep the examples more-or-less where they are and add the extra rendering pieces there? E.g 0d255599fbbe330357a5686d47ad1230ae819371

Reply:
That works for me (I had kept all "off-project" files in "_resources" mostly wishing to eventually call it "_tests". But there's no 100% clean organization either way; 0d25559 is good.)